src/analysis/special_functions/log.lean

```diff
@@ -172,6 +172,32 @@ begin
   { rintro (rfl|rfl|rfl); simp only [log_one, log_zero, log_neg_eq_log], }
 end
 
+lemma log_le_sub_one_of_nonneg {x : ℝ} (hx : 0 < x) : log x ≤ x - 1 :=
+begin
+  have h := add_one_le_exp (log x),
+  rw exp_log hx at h,
+  rw le_sub_iff_add_le,
+  exact h,
+end
+
+lemma log_div_self_decreasing {x y : ℝ} (hex : exp 1 ≤ x) (hxy : x ≤ y) :
```

Comment:
Could you please formulate a version with `antitone_on` (or better `strict_anti_on` if it doesn't make the proof much harder)?
BTW, if you don't need this fact earlier than we have derivatives, then you can just differentiate `log x / x` (not sure if it will be shorter in Lean).

Reply:
`strict_anti_on` is possible, but I think the best way to go about it would be to go back and prove more of the `exp` lemmas, so better for another PR.
WRT derivatives, we've gone back and forth so many times on the Bertrand PR on calculus vs no calculus that I'm hesitant to switch again. This result would be the only fact that requires calculus in the Bertrand tree, and why invoke the whole derivative concept when a 20-line lemma suffices?

Reply:
I propose making `strict_anti_on` a good-first-project issue after this PR is merged.